lms/product/plugin/misc.py

```diff
@@ -0,0 +1,26 @@
+class MiscPlugin:
+    """
+    Plugin for product differences that don't have a better abstractions around them.
+
+    Works as a placeholder location to centralize LMS differences here without the need to create
+    bigger abstractions before it's clear what that abstraction should look like.
+
+    The aim also includes to remove any checks around product.family around the code base.
+
+    New methods here should not try to get a very tight API as easier to refactor once multiple
+    LMS have the same issue vs getting the right parameters in the first occurrence.
+
+    Once any of these is implemented by more than one product or a group of
+    methods looks like it could belong to their own plugin it's time to refactor them out.
+    """
+
+    def post_configure_assignment(self):  # pragma: nocover
```

Comment:
This could grow to have pre/post hooks for show document/configure etc.

Only needs this one now so stopping here.